### ISSUE TITLE
Image not loading on ipad

### ISSUE THREAD

**Issue by koustab** [kthornbloom/Smoothproducts#22]
hi 
i have this issue, i have downloaded the file and uploaded to my hosting. the images are loading in every devices but not on ipad. the loading image is showing on ipad but not the others. 
here is the link of it

http://listing.aphilatelicoutlet.com/ebay/sp/

could anyone please tell me what is the solution for it?

thanks in advance
/koustab

**Comment by koustab** [comment]
I hve just checked its actually problem with the safari and the hosting. the images loading on localhost but not from the server. but on the other hand the image loading fine in iphone. 

is there any solution for it ?

**Comment by kthornbloom** [comment]
Hello, this plugin has been tested as working on an iPad. Can you try the [demo](http://kthornbloom.com/smoothproducts/) on your device? If it's working, then the problem must be something else in your case. If it's not working, please let me know what generation of iPad, which browser, and version & I'll check it out!

**Comment by koustab** [comment]
Hi Kevin 

Thanks for your reply. It was a hosting server issue, as Im using a free hosting service called 000webhost. I have tried demo, and same results. I've uploaded the demo in a different hosting and it loaded and worked fine in iPad and MacBook.

The gallery  loading fine in Firefox and iPhone from 000webhost and only not loading in iPad and safari ( MacBook ). 

Sorry about the broken link.ive changed it to other host juSt last night.

I'm Einstein when it comes to coding lol =) 

Thanks
